src/app/clients/client-stepper/client-general-step/client-general-step.component.ts

```diff
@@ -71,7 +74,7 @@ export class ClientGeneralStepComponent implements OnInit {
     this.createClientForm = this.formBuilder.group({
       'officeId': ['', Validators.required],
       'staffId': [''],
-      'legalFormId': [''],
+      'legalFormId': ['', Validators.required],
```

Comment:
Are you sure this is right? For non-entity client, the legal form cannot be required

Reply:
In both cases (Person and Entity) the Legal Form is required